cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -268,12 +269,12 @@ func (s *CMServer) Run(_ []string) error {
 	).Run()
 
 	// TODO: Enable the code belowe when horizontal pod autoscaler controller is implemented.
-	// expClient, err := client.NewExperimental(kubeconfig)
-	// if err != nil {
-	// 	glog.Fatalf("Invalid API configuration: %v", err)
-	// }
-	// horizontalPodAutoscalerController := autoscalercontroller.New(expClient)
-	// horizontalPodAutoscalerController.Run(s.NodeSyncPeriod)
+	expClient, err := client.NewExperimental(kubeconfig)
+	if err != nil {
+		glog.Fatalf("Invalid API configuration: %v", err)
+	}
+	horizontalPodAutoscalerController := autoscalercontroller.New(kubeClient, expClient)
+	horizontalPodAutoscalerController.Run(s.NodeSyncPeriod)
```

Comment:
Un-commenting & enabling this code by default is not a good idea. Please note that experimental API is disable by default, and our controller will spam logs with errors. We should leave it commented, or somehow associate it with turning on the experimental API.